src/coreclr/utilcode/executableallocator.cpp

```diff
@@ -499,9 +573,9 @@ void* ExecutableAllocator::ReserveWithinRange(size_t size, const void* loAddress
         bool isFreeBlock;
         BlockRX* block = AllocateBlock(size, &isFreeBlock);
         if (block == NULL)
-        {
-            return NULL;
-        }
+	    {
```

Comment:
Nit: The formatting looks off.

Reply:
Somehow this file contained tabs at many places for some reason